src/views/Join.vue

```diff
@@ -0,0 +1,259 @@
+<template>
+<div class = "page">
+
+    <div class = "header-container">
+        <div class = "header-group">
+            <h1 class = "header-title">Change the world and build projects with us.</h1>
+            <div class = "cta-button"> 
+                <h3> APPLY </h3>
+                <div id = "button-arrow">  </div>
+            </div>
+        </div>
+    </div>
+
+    <div class = "container-right">
+        <div class = "cr-image">
+         <img id="code-img" src="@/assets/img/code@2x.png" > 
+         </div>
+        <div class = "cr-text">
+            <h2 id = "code-h2">Make code your superpower…</h2>
+            <p  id = "code-p"> Technical leadership and support from Stanford student volunteers dedicating 5 to 10 hours per week on the project free of cost. </p>
+        </div>
+    </div>
+
+    <div class = "container-left">
+        <div class = "content-left">
+        <div class = "cl-text">
+            <h2>unless your superpower is design.</h2>
+            <p>Technical leadership and support from Stanford student volunteers dedicating 5 to 10 hours per week on the project free of cost. </p>
+        </div>
+        <div class = "cl-image">
+        <img id="design" src="@/assets/img/sketch@2x.png" />
+         <img id="bg" src="@/assets/img/ctc-background@2x.png" />
+        </div>
+        </div>
+    </div>
+
+    <div class = "container-right">
+
+         <div class = "cr-image">
+         <img id="family" src="@/assets/img/family@2x.png" > 
+         </div>
+
+        <div class = "cr-text">
+            <h2 id = "family-h2" >You’re joining a family</h2>
+            <p id = "family-p">We built a project matching platform that boosted engagement by 60%.   We’re Code the Change, a club at Stanford University that builds useful software for high-impact nonprofits.  </p>
+        </div>
+    </div>
+
+    <div class = "cta">
+        <h1> Ready to jump in?  Apply today. </h1>
+        <div class = "cta-button" id ="footer-cta"> 
+                    <h3> Apply </h3>
+                     <div id = "button-arrow">
+                    </div>
+        </div>
+    </div>
+
+</div>
+
+</template>
+<script>
+export default {
+}
+</script>
+
+<style lang="scss" scoped>
+@import '../theme.scss';
+
+.page{
+}
+
+.header-container{
+  width: 100%;
+  height: 55vh;
+  background-color: $secondary-light;
+
+}
+
+.header-group{
+  font-family: OpenSans;
+  padding-left: 10em;
+  padding-right: 10em;
+  padding-top: 8em;
+  padding-bottom: 10em;
+
+}
+
+.header-title{
+  font-family: Comfortaa;
+  font-size: 3.5em;
+    font-weight: bold;
+  line-height: 1.39;
+  letter-spacing: normal;
+  color: $secondary-medium;
+}
+.cta-button{
+  margin-top: 4em;
+  width: 21vw;
+  height: 7.5vh;
+  border-radius: 8px;
+  box-shadow: 0 4px 25px 0 rgba(252, 85, 85, 0.5);
+  background-color: $secondary-heavy;
+
+
+  &:hover{
```

Comment:
Here are some cool hover css ideas: https://1stwebdesigner.com/10-pure-css-call-action-button-sets/. Check the codepen for #10.

Also, if you're going for a more lifting vibe, check out this codepen: https://codepen.io/ashleynolan/pen/djpCG